aboutme.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="homestyle.css">
+	<meta charset="utf-8">
+  </head>
+  <body>
+  	<header class="navbar">
+	  <div id="NC"><a href="home.html">Nick Clifford</a> 
+	  </div>
+	  <div class="links">
+		  	<a id="aboutme" href="aboutme.html">About Me</a>
+		  	<a id="portfolio" href="portfolio.html">Portfolio</a>
+		 	<a id="contact" href="contact.html">Contact</a>
+	  </div>
+	  
+  	</header>
+	 <div class="aboutmeparent1">
+		 <div class="aboutmechild1">Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+		 </div>
+		 <div class="aboutmechild2">Suspendisse ut commodo urna, non aliquam tellus. Aenean tempor suscipit magna vel ultricies. Nullam quis orci posuere, pharetra turpis vel, mattis velit. Praesent sed mi interdum velit euismod iaculis sit amet sed lectus. Fusce aliquam non turpis eu finibus.
+		 </div>
+	 </div>
+	 	<div>
+	 	   <img id="kyoto" src="kyoto.jpg" alt="Kyoto after dark">
+  		</div>
+   	 <div class="aboutmeparent1">
+   		 <div class="aboutmechild2">Suspendisse ut commodo urna, non aliquam tellus. Aenean tempor suscipit magna vel ultricies. Nullam quis orci posuere, pharetra turpis vel, mattis velit. Praesent sed mi interdum velit euismod iaculis sit amet sed lectus. Fusce aliquam non turpis eu finibus.
+   		 </div>
+   		 <div class="aboutmechild1">Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+   		 </div>
+   	 </div>
```

Comment:
I'm not in love with these class names! Can you come up with something more semantic for these sections?

Reply:
Switched in aboutmecontainer and aboutmecomponent1 and aboutmecomponent2.

Hope that makes more sense.